misk-admin/build.gradle.kts

```diff
@@ -18,7 +18,7 @@ dependencies {
   api(project(":wisp:wisp-config"))
   api(project(":wisp:wisp-deployment"))
   api(project(":misk"))
-  api(project(":misk-action-scopes"))
+  api(project(":misk-api"))
```

Comment:
If `misk-action-scopes` already includes `misk-api` then is this necessary/correct?

Reply:
This is addressing an issue reported by the dependency-analysis plugin Essentially misk-admin main (non-test) code now only uses classes from misk-api, and not misk-action-scopes and dependency analysis wants us to be explicit about that dependency rather than relying on a transtivive dep.